src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,38 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <div className="card" data-cy="Movie" key={movie.imdbId}>
```

Comment:
here you dont need key because you wrote it already in the parent component (key has to be in parent component when mapping)